doc/releasenotes/16_0_0_summary.md

```diff
@@ -3,27 +3,27 @@
 ### Table of Contents
 
 - **[Major Changes](#major-changes)**
-  - **[VReplication](#vreplication)**
-    - [VStream Copy Resume](#vstream-copy-resume)
-    - [VDiff2 GA](#vdiff2-ga)
-  - **[Tablet throttler](#tablet-throttler)**
-  - **[Incremental backup and point in time recovery](#inc-backup)**
-  - **[Replication manager removal and VTOrc becomes mandatory](#repl-manager-removal)**
   - **[Breaking Changes](#breaking-changes)**
     - [VTGate Advertised MySQL Version](#advertised-mysql-version)
     - [Default MySQL version on Docker](#default-mysql-version)
     - [Running Vitess on the Operator](#running-vitess-on-the-operator)
+    - [`lock-timeout` and `remote_operation_timeout` Changes](#lock-timeout-introduction)
```

Comment:
which binaries do these flags go on?

Reply:
Those are topo flags, they are used by several binaries: `vtgate`, `vttablet`, `vtctld`, `vtbackup`, `vtgr`, `vtorc`, and `vtexplain`.